CreeDictionary/API/search/ranking.py

```diff
@@ -1,92 +1,44 @@
 from __future__ import annotations
 
-from functools import cmp_to_key, partial
-from typing import Callable, Any, cast
+from math import log
 
-from utils import get_modified_distance
-from . import core
-from .query import CvdSearchType
 from .types import Result
-from ..models import wordform_cache
+from ..models import Wordform
 
 
-def sort_search_result(
-    res_a: Result,
-    res_b: Result,
-    search_run: core.SearchRun,
-) -> float:
-    """
-    determine how we sort search results.
+def _has_value(value):
+    if value is None:
+        return 0
+    return 1
```

Comment:
Please don't change anything here but...

did you know...

that `isinstance(True, bool)`?

...and that `0 == False` and `1 == True`

it's... uh, yep.

Uh huh.